lib/Doctrine/ORM/Mapping/ReflectionEnumProperty.php

```diff
@@ -56,32 +58,54 @@ public function getValue($object = null)
             return null;
         }
 
+        if (is_array($enum)) {
+            return array_map(static function (BackedEnum $item): mixed {
+                return $item->value;
+            }, $enum);
+        }
+
         return $enum->value;
     }
 
     /**
-     * @param object $object
-     * @param mixed  $value
+     * @param object     $object
+     * @param int|string $value
```

Comment:
This should silence PHPStan:

```suggestion
     * @param object                    $object
     * @param int|string|int[]|string[] $value
```